index.html

```diff
@@ -0,0 +1,26 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Booklopedia</title>
+    <link rel="stylesheet" href="style.css" />
```

Comment:
your file name is `home.css` not `style.css`

```suggestion
    <link rel="stylesheet" href="home.css" />
```